packages/selinux-policy/fs.cil

```diff
@@ -43,6 +43,10 @@
 (filecon "/.*/usr/bin/containerd.*" file runtime_exec)
 (filecon "/.*/usr/bin/docker.*" file runtime_exec)
 (filecon "/.*/usr/sbin/runc" file runtime_exec)
+(filecon "/.*/usr/bin/apiserver" file api_exec)
```

Comment:
Would the "all_s" rule in combination with this `.*` wildcard allow someone to create/mount `/local/usr/bin/apiserver` and have it get `api_exec` and `api_t` and bypass our restrictions?

Reply:
These statements are used to generate `file_contexts`, which is a text file used by tools like `setfiles`.

This is one of the reasons that the "relabel" actions are all reserved (though still allowed for `super_t`). However, you don't need `file_contexts` from the host to perform the relabel if you're allowed to relabel; it's just a text file and you can write your own with the patterns you want.